src/lib/generator/writeCsv.js

```diff
@@ -1,24 +1,30 @@
 import fs from 'fs';
-import { join, dirname, resolve } from 'path';
+
 import _ from 'lodash';
 
-const __dirname = resolve(dirname(''));
+import { icdCodesJsonFilePath, icdCodesCsvFilePath } from '.';
 
-function writeCsvLibrary() {
-    /**@type {Array<App.CodeEntry>} */
-    const jsonLibrary = JSON.parse(
-        fs.readFileSync(join(__dirname, 'src/files/output/icd_codes.json'), 'utf8'),
-    );
+/**
+ * @param {App.CodeEntry[]} jsonLibrary
+ */
+export function mapCsv(jsonLibrary) {
     const head = 'code,description,hcc_v24,hcc_v28,is_specific,parent,children';
     const rows = _.map(jsonLibrary, (entry) => {
         const parent = entry.parent ? entry.parent.code : '';
         const children = entry.children ? _.map(entry.children, 'code') : '';
-        return `${entry.code},"${entry.description}",${entry.hcc_v24},${entry.hcc_v28},${entry.is_specific},${parent},${children},`;
+        return `${entry.code},"${entry.description}",${entry.hcc_v24||''},${entry.hcc_v28||''},${entry.is_specific},${parent},${children},`;
```

Comment:
I think `|| ''` is better than `undefined` for the `hcc_v24` and `hcc_v28` columns